components/license-control-service/pkg/storage/storage.go

```diff
@@ -308,9 +323,9 @@ func (p *PGBackend) StoreDeployment(ctx context.Context, id string) error {
 			}
 		} else {
 			_, err := tx.Exec(
-				`INSERT INTO deployment (id, created_at, updated_at)
-				VALUES ($1, $2, $2)`,
-				id, nowTime)
+				`INSERT INTO deployment (id, created_at, updated_at, type_id)
```

Comment:
Should we not add it in update step also in line 318

Reply:
Kept it intentionally so that even if the user removes the folder and restarts service later, it should not make it a Standalone